projects/Amlogic_GX/options

```diff
@@ -0,0 +1,146 @@
+################################################################################
+# setup system defaults
+################################################################################
+
+  # The TARGET_CPU variable controls which processor should be targeted for
+  # generated code.
+    case $TARGET_ARCH in
+      aarch64)
```

Comment:
Would be nice to have support for 32bit userspace like C2, WH, WP2 already have in master:
https://github.com/LibreELEC/LibreELEC.tv/commit/de9d1af69f2e25289be3516fa5b35b238e430ea3#diff-ed30c2b16c5c3d9c1884a81bef426347R24

Reply:
It should be worth trying

Reply:
32bit is working fine now !